tests/system/test_session_api.py

```diff
@@ -2517,6 +2517,141 @@ def test_partition_query(sessions_database, not_emulator):
     batch_txn.close()
 
 
+def test_should_close_inactive_transactions_with_bursty_pool_and_transaction(
```

Comment:
Can we add tests that for PDML/partitioned reads we shouldn't be closing inactive sessions?